### PR TITLE
FIX: correct font-sizes (again!)

### PR DESCRIPTION
**Pull request by agoose77**

### CODE REVIEW

**Comment by agoose77** [comment]
@rowanc1 can I get a quick sense check on the use of preflight.css vs the tailwind config?

**Comment by agoose77** [comment]
please update snapshots

File: tailwind.config.js [hunk 1]
```diff
@@ -20,11 +20,11 @@ module.exports = {
       typography: theme => ({
         DEFAULT: {
           css: {
+            fontSize: 'var(--jp-content-font-size1)',
```
Comment (rowanc1) [inline tailwind.config.js:23]: Does this need to be protected in a `.myst`? Or this is in the typography anyways (I think that is the case..!).

**Comment by rowanc1** [comment]
I think overall this makes sense! It would be nice to have a typography snapshot to see and predict these changes as well...!

File: tailwind.config.js (hunk 1 above)
Comment (agoose77) [reply tailwind.config.js:23]: Typography has
```
.myst {
  @apply prose prose-stone dark:prose-invert break-words max-w-none w-full overflow-auto;
}
```

Which, as I understand it, is where the MyST stylings are actually applied

**Comment by agoose77** [comment]
please update snapshots

**Comment by agoose77** [comment]
please update snapshots

**Comment by agoose77** [comment]
please update snapshots

**Pull request merged**